helix-core/src/main/java/org/apache/helix/model/InstanceConfig.java

```diff
@@ -505,6 +498,46 @@ public void setMaxConcurrentTask(int maxConcurrentTask) {
     _record.setIntField(InstanceConfigProperty.MAX_CONCURRENT_TASK.name(), maxConcurrentTask);
   }
 
+  /**
+   * Get the instance capacity information from the map fields
+   * @return data map if it exists, or empty map
+   */
+  public Map<String, Integer> getInstanceCapacityMap() {
+    Map<String, String> capacityData = _record.getMapField(InstanceConfigProperty.INSTANCE_CAPACITY_MAP.name());
+    Map<String, Integer> capacityDataMap = new HashMap<>();
+
+    if (capacityData != null && !capacityData.isEmpty()) {
+      for (String key: capacityData.keySet()) {
```

Comment:
Does Hexli format style need a space before the ":"? I think it would be good to lint/format the code using Helix plugin.

Reply:
Good catch.